docs/reference/esql/metadata-fields.asciidoc

```diff
@@ -17,20 +17,24 @@ supported ones are:
   * `_version`: the source document's version. The field is of the type
   <<number,long>>.
 
-To enable the access to these fields, the <<esql-from,`FROM`>> source command needs
-to be provided with a dedicated directive:
+  * <<mapping-source-field,`_source`>>: the original JSON document body 
+  that was passed at index time. The field is loaded as a special `_source` 
+  type. This field is not supported by functions.
+
+To enable access to these fields, the <<esql-from,`FROM`>> source command 
+requires a dedicated directive:
 
 [source,esql]
 ----
-FROM index [METADATA _index, _id]
+FROM index [METADATA _index, _id, _source]
```

Comment:
Note that a syntax change is coming in https://github.com/elastic/elasticsearch/pull/105221, which may require a follow-up PR.